src/geo/map-view.js

```diff
@@ -157,12 +157,12 @@ var MapView = View.extend({
 
   /** unbind model properties */
   _unbindModel: function () {
-    this.map.unbind('change:view_bounds_sw', this._changeBounds, this);
-    this.map.unbind('change:view_bounds_ne', this._changeBounds, this);
-    this.map.unbind('change:zoom', this._setZoom, this);
-    this.map.unbind('change:scrollwheel', this._setScrollWheel, this);
-    this.map.unbind('change:keyboard', this._setKeyboard, this);
-    this.map.unbind('change:center', this._setCenter, this);
+    this.map.unbind('change:view_bounds_sw', null, this);
```

Comment:
Was the fact that we're specifying a callback the culprit of this?

```
 If no callback is specified, all callbacks for the event will be removed
```

Did you make sure no other objects should keep listening to these events?